.travis.yml

```diff
@@ -6,5 +6,5 @@ jdk:
 before_script:
   - mysql -e "create database IF NOT EXISTS cromwell_test;"
 
-script: sbt -Dbackend.shared-filesystem.localization.0=copy clean coverage nodocker:test
+script: sbt -Dbackend.shared-filesystem.localization.0=copy clean coverage nodocker:test assembly
```

Comment:
Is this so we can see these problems earlier?  Do you think it'll slow down tests a lot?